src/package/nuspec/Microsoft.NET.Test.Sdk.props

```diff
@@ -13,6 +13,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
+    <TestProject>true</TestProject>
     <IsTestProject>true</IsTestProject>
```

Comment:
TestProject imports all the dependencies from targets. IsTestProject marks the project as test project so you can run it via dotnet test .csproj. This was done automatically in all projects that have <TestProject>true</TestProject>